syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -50,25 +50,24 @@ def __init__(
                 which this tensor should be searchable for
             description: an optional string describing the purpose of the
                 tensor
+        Raises:
+            ValueError: Invalid dtype argument (`long` and `int` only)
+            ValueError: Shares is not a dict (optionally wrapped)
```

Comment:
Can we also support int32, int64 and the associated torch dtypes? https://pytorch.org/docs/stable/tensor_attributes.html#torch.torch.dtype

Reply:
To be more flexible for new users